README.md

```diff
@@ -1,88 +1,70 @@
 # stylelint-order [![Build Status][ci-img]][ci] [![npm version][npm-version-img]][npm] [![npm downloads last month][npm-downloads-img]][npm]
 
-A plugin pack of order related linting rules for [stylelint]. Every rule support autofixing (`stylelint --fix`).
+A plugin pack of order-related linting rules for [stylelint]. Every rule support autofixing (`stylelint --fix`).
 
 ## Installation
 
-First, install [stylelint]:
+Install [stylelint] & `stylelint-oder`:
 
 ```
-npm install stylelint --save-dev
-```
-
-Then install plugin:
-
-```
-npm install stylelint-order --save-dev
```

Comment:
I would like to keep them separate, because users most likely have stylelint installed already. So I think it's better to separate stylelint installation from stylelint-order installation.